aim/tests/unit/test_aim_manager.py

```diff
@@ -713,6 +712,42 @@ class TestBridgeDomainMixin(object):
     res_command = 'bridge-domain'
 
 
+class TestInfraMixin(object):
+    resource_class = resource.Infra
+    resource_root_type = resource.Infra._aci_mo_name
+    test_identity_attributes = {}
+    test_required_attributes = {}
+    test_search_attributes = {}
+    test_update_attributes = {}
+    test_default_values = {}
+    res_command = 'infra'
+    test_dn = 'uni/infra'
+
+
+class TestNetflowVMMExporterPolMixin(object):
+    resource_class = resource.NetflowVMMExporterPol
+    resource_root_type = resource.Infra._aci_mo_name
+    test_identity_attributes = {'name': 'netflow1'}
+    test_required_attributes = {'name': 'netflow1',
+                                'dst_addr': '172.28.184.76',
+                                'dst_port': '2055',
+                                'src_addr': '1.2.2.2',
+                                'ver': 'v9'}
+    test_search_attributes = {'name': 'netflow1'}
+    test_update_attributes = {'dst_addr': '172.28.184.77',
+                              'dst_port': '2056',
+                              'src_addr': '1.1.2.2',
```

Comment:
we should make src_addr and ver different also here.

Reply:
addressed all the comments in the latest commit